lib/transbank/sdk/patpass/patpass_by_webpay/errors/patpass_by_webpay_errors.rb

```diff
@@ -0,0 +1,12 @@
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_create_error'
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_commit_error'
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_status_error'
+
```

Comment:
You have to `require` the `integration_type_error` or you'll get  an undefined class error

Reply:
The namespace that I used for this error type is the following `Transbank::Patpass::Errors::IntegrationTypeError`, however you've got the reason, it is needed due the nested modules namespace says that.